### ISSUE TITLE
CORS request rejected 

### ISSUE THREAD

**Issue by arthurlutz** [cozy/cozy-mobile#88]
The application is failing to sync and giving me the following error message : 

```
CORS request rejected : https://cozy.example.org/cozy/_changes?descending=true&limit=1
```

When using my cozy instance via the web, I don't use ```/cozy/``` at the end of the URL, is that the problem ? In my apache log I see some 

```
"PUT /cozy/ID-HASH/file?rev=1-hash HTTP/1.1" 201 602
```

**Comment by clochix** [comment]
I'm really sorry @arthurlutz, I forgot to give you an answer.
@jacquarg, are you aware of problems with CORS requests?

**Comment by khertan** [comment]
Same error here ...

**Comment by jacquarg** [comment]
Hi,
I encounter this kind of error during development when connectivity is missing on my testing device. So this cors error may hide a more generic network error.

Could you try to send me some log from the mobile app (menu --> configuration --> send log ) if the app go up to this step ?

**Comment by khertan** [comment]
Hi

Nice feature the send log ... i should include something similar in my apps :+1: 
Logs sent

Regards

**Comment by jacquarg** [comment]
Thanks @khertan !

I received your log, and I didn't understand very well in which state was the application ? Did you reach the folder list once ?
But what is more readable in log is that there were no wifi just before the error.

Thanks about the log, re-use it as much as you want : https://github.com/cozy/cozy-mobile/blob/master/www-src/app/lib/persistent_log.coffee . It's hackish and designed for in-browser apps, but it works, and it let the user know what he is sending !

**Comment by poupotte** [comment]
This issue should be fixed with new version : 0.1.19.

**Comment by khertan** [comment]
Works for me with last release too.
